.github/workflows/ci.yml

```diff
@@ -118,7 +118,7 @@ jobs:
           $MAVEN failsafe:integration-test failsafe:verify -B --strict-checksums -P ci -pl :trino-jdbc
       - name: Clean Maven Output
         run: $MAVEN clean -pl '!:trino-server,!:trino-cli'
-      - uses: docker/setup-qemu-action@v3
+      - uses: docker/setup-qemu-action@68827325e0b33c7199eb31dd4e31fbe9023e06e3 # v3
```

Comment:
Can we enable dependabot to check if there are new versions that we might want to use? We already run it for actions we copied in https://github.com/trinodb/github-actions/, I hope it would pick up any changes we make there too.

Reply:
I don't know the answer to this question.

Reply:
https://github.com/trinodb/trino/pull/22378